lib/Target/SubstraitPB/Import.cpp

```diff
@@ -141,8 +141,29 @@ static mlir::FailureOr<CrossOp> importCrossRel(ImplicitLocOpBuilder builder,
   return builder.create<CrossOp>(leftVal, rightVal);
 }
 
-static mlir::FailureOr<UnionDistinctOp>
-importSetRel(ImplicitLocOpBuilder builder, const Rel &message) {
+std::optional<SetOpKind> getSetOpKind(SetRel::SetOp kind) {
+  switch (kind) {
+  case ::substrait::proto::SetRel::SET_OP_UNSPECIFIED:
+    return SetOpKind::unspecified;
+  case ::substrait::proto::SetRel::SET_OP_MINUS_PRIMARY:
+    return SetOpKind::minus_primary;
+  case ::substrait::proto::SetRel::SET_OP_MINUS_MULTISET:
+    return SetOpKind::minus_multiset;
+  case ::substrait::proto::SetRel::SET_OP_INTERSECTION_PRIMARY:
+    return SetOpKind::intersection_primary;
+  case ::substrait::proto::SetRel::SET_OP_INTERSECTION_MULTISET:
+    return SetOpKind::intersection_multiset;
+  case ::substrait::proto::SetRel::SET_OP_UNION_DISTINCT:
+    return SetOpKind::union_distinct;
+  case ::substrait::proto::SetRel::SET_OP_UNION_ALL:
+    return SetOpKind::union_all;
+  default:
+    return std::nullopt;
+  }
+}
```

Comment:
We don't call this function in the end, right? Then we should remove it, no?

Reply:
We use this in the import function. The one we discussed/I removed was in the export function.

Reply:
Ah, OK, now I was too quick... I think the static cast should work the other way as well. Can you try?

Reply:
On it.